.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,10 @@
+Thank you for your pull request. In order for your pull request to be accepted please check the following.
+
+- [ ] I have Contribution Guidelines at https://github.com/syl20bnr/spacemacs/tree/develop/CONTRIBUTING.org
+- [ ] My pull request is against the `develop` branch and not `master`
+- [ ] I have rebased my contribution against `develop` and not `master`
```

Comment:
I think it might be not clear to users. Though not sure what is the best way to say this :D